attacker/attacker.go

```diff
@@ -89,16 +89,17 @@ func indexVegetaResults(ctx context.Context, metrics vegeta.Metrics, testName st
 		RunID:          attackMap["RUNID"],
 	}}, indexers.IndexingOpts{})
 	if err != nil {
-		return fmt.Errorf(err.Error())
+		return err
 	} else {
-		zlog.Debug(ctx).Msg(resp + "\n")
+		zlog.Info(ctx).Msg(resp)
```

Comment:
What is the point in the `else` here?

Reply:
Great, but as I originally suggested you don't need the `else` I don't think, just log after the `return err`